app/routes/irrigated-land.js

```diff
@@ -25,7 +25,7 @@ function createModel (irrigatedLandCurrent, irrigatedLandTarget, errorMessageLis
         text: 'ha'
       },
       hint: {
-        text: 'Enter figure in hectares'
+        html: '<font color="black"> Hectares </font><br/> Enter figure in hectares, for example 543.5'
```

Comment:
html: '<span class="govuk-label govuk-!-font-weight-bold"> Hectares </span>Enter figure in hectares, for example 543.5'